skopt/optimizer/optimizer.py

```diff
@@ -151,7 +155,14 @@ def __init__(self, dimensions, base_estimator,
                 self._cat_inds.append(ind)
             else:
                 self._non_cat_inds.append(ind)
-        self._check_arguments(base_estimator, n_random_starts, acq_optimizer)
+
+        if n_random_starts is not None:
+            warnings.warn(("n_random_starts has been renamed to "
```

Comment:
It's not just a rename right. Maybe we can rename this to `n_random_starts will be removed. Use n_initial_points instead`?